src/components/sections/ProductDetails/ProductDetails.tsx

```diff
@@ -34,13 +34,13 @@ function ProductDetails({ product: staleProduct }: Props) {
       brand,
       isVariantOf,
       image: productImages,
-      offers,
+      offers: { offers },
     },
   } = data
 
-  const price = offers.offers[0]?.price
-  const listPrice = offers.offers[0]?.listPrice
-  const seller = offers.offers[0]?.seller
+  const price = offers[0]?.price
+  const listPrice = offers[0]?.listPrice
+  const seller = offers[0]?.seller
```

Comment:
Can you explain the motivation behind those changes? IMO, it seems to be the same, semantically and for readability.

Reply:
The motivation was just to improve reading.